ident/key.php

```diff
@@ -171,22 +171,22 @@ function openGlossaryPopup(glossid){
 if($dynClid){
 	if($dataManager->getClType() == 'Specimen Checklist'){
 		echo '<a href="'.$CLIENT_ROOT.'/collections/list.php?tabindex=0">';
-		echo 'Occurrence Checklist';
+		echo $LANG['OCC_CHECKLIST'];
 		echo '</a> &gt;&gt; ';
 	}
 }
 elseif($clid){
 	echo '<a href="'.$CLIENT_ROOT.'/checklists/checklist.php?clid='.$clid.'&pid='.$pid.'">';
-	echo 'Checklist: '.$dataManager->getClName();
+	echo $LANG['CHECKLIST'].': '.$dataManager->getClName();
```

Comment:
In general, per the Symbiota coding standards doc, we're recommending:
> add spaces before and after periods when concatenating in php

Reply:
So, in this case , 
`$LANG['CHECKLIST'] . ': ' . $dataManager->getClName();` instead of
`$LANG['CHECKLIST'].': '.$dataManager->getClName();`

Reply:
@Atticus29 , thanks for catching that! It looks like the spacing wasn't consistently that way, so I went through and tried to make it consistently spaced.